.changeset/two-geese-eat.md

```diff
@@ -0,0 +1,24 @@
+---
+'astro': major
```

Comment:
Hm, somehow got 2 changesets pointing to astro core?

Reply:
I think this is correct as this changeset is a breaking change for the `astro:build:start` hook 🤔  The other breaking change is for Astro globals which I thought would be good to separate.

Reply:
Okay, if that separation resolves then perfect 👍